### ISSUE TITLE
input.value always null

### ISSUE THREAD

**Issue by GrishinSergeyAl** [rjaros/kvision#432]
Hello!

I can't get the value from the input

```
class App : Application() {
    private val observable = ObservableValue(0).also {
        it.subscribe {
            console.log("observable is $it")                            // never comes here
        }
    }

    override fun start(state: Map<String, Any>) {
        root("kvapp") {
            input( InputType.NUMBER){
                bind(observable){
                    console.log("new value is $it")                      // never comes here
                }
                onEvent {
                    click = {
                        console.log("oldValue is ${this@input.value}")   // oldValue always is null
                    }
                }
            }
        }
    }
}

javaVersion=1.8
#Plugins
systemProp.kotlinVersion=1.7.0
serializationVersion=1.4.0
#Dependencies
systemProp.kvisionVersion=5.14.0
ktorVersion=2.1.0
hikariVersion=3.2.0
commonsCodecVersion=1.10
jdbcNamedParametersVersion=1.1
logbackVersion=1.2.3
exposedVersion=0.39.2
pgsqlVersion=42.2.2

```

**Comment by rjaros** [comment]
Hello!

Perhaps it's a bit non-intuitive, but the `io.kvision.html.Input` component is not an interactive form component (like e.g. `Text`, `Select` or `Spinner`). This is just a simple wrapper for HTML INPUT element, and its `value` property is not automatically bound to the internal value of the element. It was added to support rendering any type of input elements. 

Generally if you want to have numeric input, you should probably use full-featured `Spinner` component from the `kvision-bootstrap-spinner` module.

But as KVision is very flexible, you can also make `Input` component work for your use-case. Just add a handler for `change` event, using code like this:
```Kotlin
val observable = ObservableValue(5)

input(InputType.NUMBER) {
    bind(observable) {
        console.log("new value is $it")                      // new value
        value = it.toString()                                // update property from observable
    }
    onEvent {
        change = {
            observable.value = it.target.asDynamic().value.toString().toIntOrNull() ?: 0  // update observable
        }
        click = {
            console.log("old value is ${this@input.value}")   // old value
        }
    }
}
```

**Comment by GrishinSergeyAl** [comment]
Thanks! 
I made my class based on TechInput, in which instead of **attributeSetBuilder.add("type", type.type)** I did **attributeSetBuilder.add("type", "number")**
Maybe you should add  **NUMBER("number")** in **enum class TextInputType** ?

**Comment by rjaros** [comment]
I should probably add all missing values.

**Comment by rjaros** [comment]
Released with 5.15.0
